gatein/src/main/java/org/exoplatform/platform/qa/ui/gatein/pageobject/UserAndGroupManagement.java

```diff
@@ -540,11 +540,8 @@ public void editUserInfo_UserProfileTab(String givenName,
     if (language != null && language != "") {
       evt.select(ELEMENT_LANGUAGE, language);
     }
-    evt.click(ELEMENT_SAVE_BUTTON);
-    evt.waitForElementNotPresent(ELEMENT_SAVE_BUTTON);
-    evt.waitForMessage(ELEMENT_MSG_UPDATE_USER_PROFILE);
-    evt.click(GateinLocator.ELEMENT_CLOSE_MESSAGE);
-    evt.waitForElementNotPresent(GateinLocator.ELEMENT_CLOSE_MESSAGE);
+    (ELEMENT_SAVE_BUTTON).click();
+    (ELEMENT_MSG_UPDATE_USER_PROFILE).click();
```

Comment:
1-evt.waitForElementNotPresent(ELEMENT_SAVE_BUTTON); ==> is not migrated  
2-evt.waitForMessage(ELEMENT_MSG_UPDATE_USER_PROFILE); become $(ELEMENT_MSG_UPDATE_USER_PROFILE).waitUntil(visible,Configuration.timeout); 
3-evt.click(GateinLocator.ELEMENT_CLOSE_MESSAGE);  ==> is not migrated
4-evt.waitForElementNotPresent(GateinLocator.ELEMENT_CLOSE_MESSAGE);   ==> is not migrated